iridium-client-nextjs/src/lib/services/cookie.service.ts

```diff
@@ -0,0 +1,28 @@
+export class CookieService {
+    public getCookie(name: string) {
+        let ca: Array<string> = document.cookie.split(';');
+        let caLen: number = ca.length;
+        let cookieName = `${name}=`;
+        let c: string;
```

Comment:
The let keyword here typically lets others know that this variable is going to change somewhere down the line. These all look like they're assigned once and that is it. I'd change these to const, and try and make the naming a little more intuitive due to the open source nature of this project. I'm not sure what ca, c, caLen mean